apis/v1alpha1/opentelemetrycollector_webhook.go

```diff
@@ -109,5 +109,16 @@ func (r *OpenTelemetryCollector) validateCRDSpec() error {
 		}
 	}
 
+	// validate autoscale with horizontal pod autoscaler
+	if r.Spec.MaxReplicas != nil {
+		if r.Spec.MaxReplicas == nil || *r.Spec.MaxReplicas < int32(1) {
```

Comment:
`r.Spec.MaxReplicas == nil ` can be removed

Reply:
updated